act-rules-format/act-rules-format.bs

```diff
@@ -224,8 +224,7 @@ Rules that can be used to determine if an accessibility requirement is *satisfie
 
 A secondary accessibility requirement is a requirement that is correlated with the rule, but for which the rule is not designed to test. The outcome of the rule impacts the result of the accessibility requirement, but the rule is not intended to test the conformance of that requirement. This correlation often results in some of the rule's test cases not satisfying the secondary accessibility requirement.
```

Comment:
```suggestion
A secondary accessibility requirement is a requirement that is correlated with the rule, but which the rule is not designed to test. The outcome of the rule impacts the result of the accessibility requirement, but the rule is not intended to test the conformance of that requirement. This correlation often results in some of the rule's test cases not satisfying the secondary accessibility requirement.
```

Reply:
`for which` seems right to me.

Reply:
I _think_ including the "for" is more grammatically correct.